File: Backjoon/Backjoon2609/Backjoon2609.java
package Backjoon2609;

import java.io.BufferedReader;
import java.io.IOException;
import java.io.InputStreamReader;
import java.util.StringTokenizer;

public class Backjoon2609{
    public static int gcd(int num1, int num2){
        if(num2 == 0) return num1;
        else return gcd(num2, num1 % num2);
    }
    public static void main(String args[]) throws IOException{
        BufferedReader br = new BufferedReader(new InputStreamReader(System.in));
        StringTokenizer st = new StringTokenizer(br.readLine());
        int A = Integer.parseInt(st.nextToken());
        int B = Integer.parseInt(st.nextToken());
        int gcd = gcd(A, B);
        System.out.println(gcd);
        System.out.print(A * B / gcd);
    }
}

File: Backjoon/Backjoon2609/Backjoon2609.c
#include <stdio.h>

int gcd(int num1, int num2){
    if(num2 == 0) return num1;
    else return gcd(num2, num1 % num2);
}

int main(){
    int A = 24;
    int B = 18;
    int g = gcd(A, B);
    printf("%d\n", g);
    printf("%d", A * B / g);
}